vendor/knative.dev/pkg/controller/stats_reporter.go

```diff
@@ -199,7 +199,7 @@ func (r *reporter) ReportReconcile(duration time.Duration, success string, key t
 		return err
 	}
 
-	// TODO skonto: fix latency histograms
-	metrics.Record(ctx, reconcileCountStat.M(1))
+	metrics.RecordBatch(ctx, reconcileCountStat.M(1),
```

Comment:
It seems all patches under `openshift/patches/*` are dropped?

Reply:
My bad, I did not re-apply those. Will fill it.